lib/dry/types/array/member.rb

```diff
@@ -48,6 +48,13 @@ def try(input, &block)
             block ? yield(failure) : failure
           end
         end
+
+        # @api public
+        #
+        # @see Definition#to_ast
+        def to_ast
```

Comment:
Here I was having doubts regarding if the better `ast` would be `[:hash, [:member, [:something, member.to_ast]]]` but I could come with a new apart from member